project-base/storefront/components/Pages/Styleguide/StyleguideCheckboxes.tsx

```diff
@@ -31,6 +32,14 @@ export const StyleguideCheckboxes: FC = () => {
                 value={false}
                 onChange={() => null}
             />
+            <Checkbox
+                disabled
+                id="checkboxDisabled"
+                label="Disabled checkbox"
+                name="Disabled checkbox"
```

Comment:
Maybe change name and label to `Disabled checked checkbox`?

Reply:
fixed